python/cudf/bindings/reduce.pyx

```diff
@@ -40,7 +43,7 @@ _SCAN_OP = {
 }
 
 
-def apply_reduce(reduction_op, col, dtype=None):
+def apply_reduce(reduction_op, col, dtype=None, ddof=1):
```

Comment:
What is `ddof`? Need to document it?

Reply:
+1, need to document what `ddof` is or remove it.

Reply:
From Pandas docs:

```
Delta Degrees of Freedom. The divisor used in calculations is N - ddof, where N represents the number of elements.
```

Reply:
Wow, that documentation from Pandas does not help me understand what this is for.

Reply:
Apparently has to do with [Bessel's correction](https://en.wikipedia.org/wiki/Bessel%27s_correction) where `n-1` is used to divide the sum instead of `n` (sample mean vs population mean). Not a stats person, so I don't fully understand.